.travis.yml

```diff
@@ -27,5 +27,5 @@ install:
 
 script:
     - if [[ $TEST_TARGET == "default" ]]; then
-        python -c "import gsw; print(dir(gsw))" ;
+        cd /tmp && python -c "import gsw; print(dir(gsw))" ;
```

Comment:
Like I suspected the `build` part was not needed, but that made me realize my mistake. I was calling from the checked source and not the installed package.